reference/constraints/Compound.rst

```diff
@@ -26,7 +26,7 @@ you can create your own named set or requirements to be reused consistently ever
     // src/Validator/Constraints/PasswordRequirements.php
     namespace App\Validator\Constraints;
 
-    use Symfony\Component\Validator\Compound;
+    use Symfony\Component\Validator\Constraints\Compound;
```

Comment:
Can you remove this one from this PR and submit it to 5.1 instead?

Reply:
Done #14433